src/component.js

```diff
@@ -56,7 +56,7 @@ Component.prototype.setState = function(update, callback) {
 	if (update == null) return;
 
 	if (this._internal) {
-		if (callback) this._renderCallbacks.push(callback);
+		if (callback) addCommitCallback(this._internal, () => callback.call(this));
```

Comment:
```suggestion
		if (callback) addCommitCallback(this._internal, callback.bind(this));
```

Reply:
@developit here is something to benchmark for you! I recall `bind()` being slower in <= Chrome 59 and being made faster in recent versions. We should check how `bind()` vs a closure behaves in other browsers.